packages/richtext-lexical/src/lexical/plugins/ClipboardPlugin/index.tsx

```diff
@@ -22,6 +46,32 @@ export function ClipboardPlugin() {
     return editor.registerCommand(
       COPY_COMMAND,
       (event) => {
+        // Handle decorator node case
+        const selection = $getSelection()
+        if ($isNodeSelection(selection)) {
+          const node = selection.getNodes()[0]
+
+          const serializedNode = node?.exportJSON() as SerializedUnknownLexicalNode
+          const deepCloneSerializedNode = JSON.parse(JSON.stringify(serializedNode))
```

Comment:
Are you sure this is necessary, since I believe exportJSON returns a new object reference?

If this is truly necessary, can you use `deepCopyObjectSimple` instead? It's faster than `JSON.parse(JSON.stringify`